crawler/plugins/cloudsight_environment.py

```diff
@@ -16,6 +16,9 @@
 class CloudsightEnvironment(IRuntimeEnvironment):
     name = 'cloudsight'
 
+    def get_feature(self):
```

Comment:
This should not be needed, specially not because if it is then it would break the other env plugins that we don't maintain. For example, the watson plugin does not have  a get_feature() method.

Features is something that only makes sense to crawler plugins (e.g. cpu, or memory).